src/Types/Comments.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comments {
```

Comment:
```suggestion
export interface Comment {
```

Reply:
Rename file too

Reply:
fixed

Reply:
mb you din't push changes? name didn't fixed

Reply:
My bad ^_^